src/app/core/services/user.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { IUser } from '../../pages/user/user.model';
+import { Observable, of } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { Users } from '../../../assets/data/users';
+
+@Injectable()
+export class UserService {
```

Comment:
Would be better to rename this service to AuthService, because this service is responsible for auth actions such as login, registration ect